front/app/containers/InitiativesEditPage/InitiativesEditFormWrapper.tsx

```diff
@@ -396,7 +397,7 @@ const InitiativesEditFormWrapper = ({
     }
   };
 
-  if (image === undefined && initiativeImage) return null;
+  const initiativeTopics = topics.data.filter((topic) => !isNilOrError(topic));
```

Comment:
Replicated what we did in the parent component with the topics before passing them through.